wdl/ONTWholeGenome.wdl

```diff
@@ -91,28 +91,44 @@ workflow ONTWholeGenome {
             bed_to_compute_coverage = bed_to_compute_coverage
     }
 
-    if (call_variants) {
+    String dir = outdir + "/alignments"
+
+    call FF.FinalizeToFile as FinalizeBam { input: outdir = dir, file = usable_bam, name = "~{participant_name}.bam" }
+    call FF.FinalizeToFile as FinalizeBai { input: outdir = dir, file = usable_bai, name = "~{participant_name}.bam.bai" }
+
+    if (defined(bed_to_compute_coverage)) { call FF.FinalizeToFile as FinalizeRegionalCoverage { input: outdir = dir, file = select_first([coverage.bed_cov_summary]) } }
+
+    ####################################################################################################
+    if (call_svs || call_small_variants) {
 
         # verify arguments are provided
-        if (! defined(fast_less_sensitive_sv)) {call Utils.StopWorkflow as fast_less_sensitive_sv_not_provided {input: reason = "Unprovided arg fast_less_sensitive_sv"}}
-        if (! defined(run_dv_pepper_analysis)) {call Utils.StopWorkflow as run_dv_pepper_analysis_not_provided {input: reason = "Unprovided arg run_dv_pepper_analysis"}}
-        if (! defined(dvp_threads)) {call Utils.StopWorkflow as dvp_threads_not_provided {input: reason = "Unprovided arg dvp_threads"}}
-        if (! defined(ref_scatter_interval_list_locator)) {call Utils.StopWorkflow as ref_scatter_interval_list_locator_not_provided {input: reason = "Unprovided arg ref_scatter_interval_list_locator"}}
-        if (! defined(ref_scatter_interval_list_ids)) {call Utils.StopWorkflow as ref_scatter_interval_list_ids_not_provided {input: reason = "Unprovided arg ref_scatter_interval_list_ids"}}
-        if (! defined(call_small_vars_on_mitochondria)) {call Utils.StopWorkflow as call_small_vars_on_mitochondria_not_provided {input: reason = "Unprovided arg call_small_vars_on_mitochondria"}}
+        if (call_svs) {
+            if (! defined(fast_less_sensitive_sv)) {call Utils.StopWorkflow as fast_less_sensitive_sv_not_provided {input: reason = "Calling SVs without specifying arg fast_less_sensitive_sv"}}
```

Comment:
This is the major section of the PR that gives me pause. In trying to apply the workflows to malaria data, I found this section to cause issues.  The problem is that Terra doesn't have a way of telling you these optional parameters are actually required.  Only the failure of the workflow reveals the fact that values must be supplied for these parameters.

Reply:
Now that it's a blocker, I'll update.
I'll default to a cheap-version (alignment + scatter-gather SV), and turn off small variants.
If one wants to turn on small variants, DV-pepper will off by-default as it's expensive, so just remember to turn that on as well if intended.